showcase-app/test-cases.md

```diff
@@ -95,3 +95,30 @@ If you want to run the tests interactively, you use `npm run test:ui`.
   - Reload the page.
   - Verify that the active profile is still displayed.
 
+### 11. Login without extension using WebID
+
+- **Description**: Test the login flow using a WebID input field.
+- **Test Steps**:
+  - Register a profile.
+  - Load main page.
+  - Enter the profile's WebID in the input field.
+  - Log in using profile's credentials.
+  - Verify that upon redirect to the main page, the app is authenticated.
+
+### 12. Displays logged in profile's WebID when available
+
+- **Description**: Test if the profile's name is displayed when it's returned from the identity provider
```

Comment:
The identity provider normally doesn't return the name of a profile. Also the provider doesn't know about profiles.

Reply:
maybe "profile" is not the right word.
the plugin was already checking for a "name" property on the returned identity, so I assumed that.
What name were you referring to here: https://github.com/SolidLabResearch/solid-identity-manager/issues/9#issuecomment-1828072113

Reply:
We assume that the name is in the WebID.

Reply:
what part of the WebID can we assume to be the name?
e.g. for the playground I can register the following WebID:
https://pod.playground.solidlab.be/svdwyer/profile/card#me
-> can we assume that the first part of the path following the domain is the name?
as in: https://somedomain.ext/<name>/rest/of/path

Reply:
ok, that's also how it's implemented. So it's just the description of the test case that needs to be adjusted: don't use "profile", instead use "webid"?